lib/errors/IdentityAssetLockTransactionOutputNotFoundError.js

```diff
@@ -0,0 +1,21 @@
+const ConsensusError = require('./ConsensusError');
+
+class IdentityAssetLockTransactionOutputNotFoundError extends ConsensusError {
+  /**
+   * @param {number} outputIndex
+   */
+  constructor(outputIndex) {
+    super(`Asset Lock Transaction Output with index ${outputIndex} not found`);
```

Comment:
Not sure if we should have an input data in the message, otherwise looks alright to me.

Reply:
Yeah, because of isolates. This one is ok, but `object.property` won't work.